docs/stdlib/abstract.rst

```diff
@@ -4,8 +4,8 @@
 Abstract Types
 ==============
 
-These types are used in definitions of polymorphic (generic) functions
-that can be applied to a broad range of types.
+Abstract types are used to describe a polymorphic, otherwise known as
+generic function to be applicable over a broad range of types.
```

Comment:
Grammar nerd here. Since we're talking about "types" (plural), I think we should say that they describe "polymorphic function**s**" (also plural) rather than "a polymorphic function" (singular)

Also, I would add another comma after "function" to fully set off that phrase from the rest of the sentence.

Suggested rewrite:
> Abstract types are used to describe polymorphic functions, otherwise known as "generic functions," which can return a broad range of types.